Pipfile

```diff
@@ -0,0 +1,8 @@
+[[source]]
+url = "https://pypi.python.org/simple"
```

Comment:
I'm like 80% sure that the source should be the source for the project requirements - not the source of the project itself. I can't find any examples of Pipfiles that don't use pypi as a source - it might be too early in its development. https://github.com/pypa/pipfile#pipfile-1

I looked through the code for pipfile and Pipenv, and I can't tell for certain. Need to find a complete spec, which I'm not certain exists right now.